#! /bin/sh /usr/share/dpatch/dpatch-run
## 80-bugfix-LOGCXX-298.dpatch by  <matthew@localhost>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Bugfix for build errors with smtp 

@DPATCH@
diff -urNad log4cxx-0.10.0~/src/main/cpp/smtpappender.cpp log4cxx-0.10.0/src/main/cpp/smtpappender.cpp
--- log4cxx-0.10.0~/src/main/cpp/smtpappender.cpp	2008-03-31 15:34:09.000000000 -0700
+++ log4cxx-0.10.0/src/main/cpp/smtpappender.cpp	2008-07-17 06:39:34.000000000 -0700
@@ -496,7 +496,7 @@
    activate &= asciiCheck(bcc, LOG4CXX_STR("bcc"));
    activate &= asciiCheck(from, LOG4CXX_STR("from"));
  
-#if !LOG4CXX_HAS_LIBESMTP
+#if !LOG4CXX_HAVE_LIBESMTP
    LogLog::error(LOG4CXX_STR("log4cxx built without SMTP support."));
    activate = false;
 #endif     
@@ -598,7 +598,7 @@
 */
 void SMTPAppender::sendBuffer(Pool& p)
 {
-#if LOG4CXX_HAS_LIBESMTP
+#if LOG4CXX_HAVE_LIBESMTP
    // Note: this code already owns the monitor for this
    // appender. This frees us from needing to synchronize on 'cb'.
    try
